game/world/managers/objects/units/player/StatManager.py

```diff
@@ -415,6 +417,19 @@ def update_base_mana_regen(self):
         regen = CLASS_BASE_REGEN_MANA[player_class] + spirit * CLASS_SPIRIT_SCALING_MANA[player_class]
         self.base_stats[UnitStats.POWER_REGENERATION_PER_5] = int(regen / 2)
 
+    def update_base_melee_critical_chance(self):
+        if self.unit_mgr.get_type_id() != ObjectTypeIds.ID_PLAYER:
+            return
```

Comment:
Should creatures be able to update their crit chance too?

Reply:
Creatures don't need to update their base chance, as instead of scaling off attributes they have a base crit chance.